rootfs/deis/production.py

```diff
@@ -1,6 +1,4 @@
-import os
```

Comment:
yes, that would be a lovely world to live in if we didn't have to support k8s v1.1 clusters as well, but I agree that we can eventually turn this into a separate module. It's good for now, just curious about the name change. :)